.mailmap

```diff
@@ -102,6 +105,7 @@ Casey Bodley <cbodley@redhat.com> <cbodley@redat.com>
 Casey Bodley <cbodley@redhat.com> casey Bodley <cbodley@redhat.com>
 Casey Bodley <cbodley@redhat.com> Casey <cbodley@redhat.com>
 Casey Marshall <csm@soe.ucsc.edu> <rsdio@29311d96-e01e-0410-9327-a35deaab8ce9>
+Casey Bodley <cbodley@redhat.com> <cbodley@users.noreply.github.com>
```

Comment:
Should probably be moved up one line, so that all @cbodley's entries are together.